editor/edit-post/keyboard-shortcuts.js

```diff
@@ -0,0 +1,6 @@
+export default {
+	toggleEditorMode: {
+		value: 'mod+shift+alt+m',
+		label: '⌘+Shift+Alt+M',
```

Comment:
Minor: I wonder if we should detect the platform and replace `⌘` with `ctrl` if not on MacOS.

Reply:
Is it reliable to use navigator.platform? I can't find any example in the Gutenberg code.

Reply:
I think it is but that's the first time we need it :)